accounts-db/src/accounts_update_notifier_interface.rs

```diff
@@ -24,11 +26,40 @@ pub trait AccountsUpdateNotifierInterface: std::fmt::Debug {
         &self,
         slot: Slot,
         write_version: u64,
-        account: &StoredAccountMeta,
+        account: &AccountForGeyser<'_>,
     );
 
     /// Notified when all accounts have been notified when restoring from a snapshot.
     fn notify_end_of_restore_from_snapshot(&self);
 }
 
 pub type AccountsUpdateNotifier = Arc<dyn AccountsUpdateNotifierInterface + Sync + Send>;
+
+/// Account type with only the fields necessary for Geyser
+#[derive(Debug, Clone)]
+pub struct AccountForGeyser<'a> {
+    pub pubkey: &'a Pubkey,
+    pub lamports: u64,
+    pub owner: &'a Pubkey,
+    pub executable: bool,
+    pub rent_epoch: Epoch,
+    pub data: &'a [u8],
+}
+
+impl ReadableAccount for AccountForGeyser<'_> {
```

Comment:
It seems that we can remove `ReadableAccount` traits and just use the `AccountForGeyser<'a>` directly. 
We are not passing `ccountForGeyser<'a>` by `ReadableAccount` anywhere in the code.

Reply:
ok. then we can keep it. But probably add a comment to indicate this is only used for testing scenario.